### PR TITLE
preview: fixes for VMR9; compile fix for lite versions (RAR)

### PR DESCRIPTION
**Pull request by adipose**

### CODE REVIEW

File: src/mpc-hc/FGFilter.cpp [hunk 1]
```diff
@@ -487,6 +487,21 @@ HRESULT CFGFilterVideoRenderer::Create(IBaseFilter** ppBF, CInterfaceList<IUnkno
                         pMFVDC->SetRenderingPrefs(MFVideoRenderPrefs_DoNotRepaintOnStop);
                     }
                 }
+                if (m_bIsPreview) {
```
Comment (clsid2) [inline src/mpc-hc/FGFilter.cpp]: This part was moved into the if() above.

File: src/mpc-hc/FGManager.cpp [hunk 2]
```diff
@@ -2580,7 +2580,7 @@ CFGManagerPlayer::CFGManagerPlayer(LPCTSTR pName, LPUNKNOWN pUnk, HWND hWnd, boo
                 break;
         }
     } else {
-        if (CAppSettings::IsVideoRendererAvailable(VIDRNDT_DS_EVR)) {
+        if (false && CAppSettings::IsVideoRendererAvailable(VIDRNDT_DS_EVR)) {
```
Comment (clsid2) [inline src/mpc-hc/FGManager.cpp]: Forgot to remove

File: src/mpc-hc/FGFilter.cpp (hunk 1 above)
Comment (adipose) [reply src/mpc-hc/FGFilter.cpp]: Hmm, yes that looks like duplicate code, maybe the merge?

File: src/mpc-hc/FGManager.cpp (hunk 2 above)
Comment (adipose) [reply src/mpc-hc/FGManager.cpp]: Yes, I had removed it, but most not have saved before commit.

**Pull request merged**